influxql/ast.go

```diff
@@ -1029,40 +1073,91 @@ func (s *SelectStatement) validateFields() error {
 	return nil
 }
 
+func (s *SelectStatement) allowMixedAggregates(numAggregates int) error {
```

Comment:
So I understand this.

`top(), value` is allowed.
`top(), mean(), value()` is not.
`top(), mean()` is allowed.

Is that correct?